src/inference/src/os/lin/lin_system_conf.cpp

```diff
@@ -516,28 +623,35 @@ void parse_freq_info_linux(const std::vector<std::vector<std::string>> system_in
         }
     }
 
-    if ((_sockets >= 1) && (ecore_enabled)) {
-        _sockets = 0;
-    }
+    _sockets = (_sockets > 0) ? _sockets + 1 : 1;
 
-    if (_sockets >= 1) {
-        _proc_type_table.resize(_sockets + 2, std::vector<int>(PROC_TYPE_TABLE_SIZE, 0));
-        for (int n = 0; n < _processors; n++) {
-            _proc_type_table[0][ALL_PROC]++;
-            _proc_type_table[_cpu_mapping_table[n][CPU_MAP_SOCKET_ID] + 1][ALL_PROC]++;
+    if (node_info_table.size() == 0) {
+        if ((_sockets > 1) && (ecore_enabled)) {
+            _sockets = 1;
```

Comment:
It looks not obvious. Could you please clarify why we consider hybrid system to have single socket/numa_node?

Reply:
Then such WAs should be described using comments. Otherwise it looks really confusing.

Reply:
updated